builtin/providers/aws/resource_aws_lambda_function.go

```diff
@@ -586,3 +587,14 @@ func validateRuntime(v interface{}, k string) (ws []string, errors []error) {
 	}
 	return
 }
+
+func validateTimeout(v interface{}, k string) (ws []string, errors []error) {
+	timeout := v.(int)
+
+	if timeout > 300 {
+		errors = append(errors, fmt.Errorf(
+			"Lambda timeout can't exceed 300. Currently set to %d.",
```

Comment:
I couldn't either. All I know is that I get an error back from them with a value. I'm guessing with both have accounts with AWS do you want to open up a support request to ask them about it? Or shall I?